README.md

```diff
@@ -19,18 +19,89 @@ This code is dual licensed under the MIT license and the OGL (Open Government Li
 
 The contents of this repository are protected by Crown Copyright (C).
 
-## Development
 
-### Requirements
-* make
-* nodejs v12 or v14
-* npm/yarn
-* [poetry](https://github.com/python-poetry/poetry)
+## Setup
+
+N.B. that some functioanlity requires environment variables to be set. Some of these are described lower down in the readme, whilst others can be found in [the environment variables section of this confluence page](https://nhsd-confluence.digital.nhs.uk/display/SPINE/Personal+Demographics+Service+api+setup)
+
```

Comment:
I think it's worth mentioning the APIGEE_USERNAME and APIGEE_PASSWORD that will need to be added to .bashrc after the env_variables.txt file has been copied over to .bashrc. Obviously, this applies for first time setup and if a dev is handling other repos, their implementation will need a little more thought than copying over the whole file into .bashrc.

Reply:
The README does mention APIGEE_USERNAME and APIGEE_PASSWORD at the bottom of the page in the API Proxy & Sandbox Service section. I believe it's only needed for deployment, not to get the sandbox working locally so I think it's ok in that section. Let me know if you disagree.